streaming/index.js

```diff
@@ -145,8 +147,13 @@ const startServer = async () => {
   });
 
   const app = express();
+  const trustProxy = proxyaddr.compile(
+    process.env.TRUSTED_PROXY_IP ?
+      process.env.TRUSTED_PROXY_IP.split(/(?:\s*,\s*|\s+)/) :
+      ['loopback', 'uniquelocal']
+  );
 
-  app.set('trust proxy', process.env.TRUSTED_PROXY_IP ? process.env.TRUSTED_PROXY_IP.split(/(?:\s*,\s*|\s+)/) : 'loopback,uniquelocal');
+  app.set('trust proxy', trustProxy);
```

Comment:
This introduces a new dependency here, and we should ensure that the behaviour is what we expect. We had some security reports regarding how we extract trusted IP addresses in the past and I would ensure that this is not an overlooked location where this can happen, as now this could be used as a way to insert incorrect data in the DB

Reply:
This isn't actually a new dependency, as express already includes this exact version. So we're just adding a dependency on a dependency of express, since we need the same functionality in an area where express doesn't cover.

Reply:
We could also add a check to validate if the value `isIP` — https://nodejs.org/docs/latest-v22.x/api/net.html#netisipinput

And then only set the `last_used_ip` if `isIP` returns true. I did note that the ruby code sometimes only sets `last_used_at`, and not `last_used_ip`, but I'm not sure why, I guess it's a nil value being stripped by activerecord?